src/gameobjects/text/static/Text.js

```diff
@@ -1198,6 +1198,27 @@ var Text = new Class({
             {
                 linePositionX += (textSize.width - textSize.lineWidths[i]) / 2;
             }
+            else if (style.align === 'justified')
+            {
+                var minimumLengthToApplyJustification = 0.85;
```

Comment:
Where does the 0.85 value come from?

Reply:
I guess this is some kind of ratio. I.e. if line width is less than 85% of defined width per line - there is no point of justifying it coz spaces would be too long.